deployer/src/main/java/com/percussion/rx/config/impl/PSConfigStatusMgr.java

```diff
@@ -106,9 +109,9 @@ public PSConfigStatus loadConfigStatusModifiable(long statusID) throws PSNotFoun
       PSConfigStatus cfgStatus = null;
       Session session = getSession();
 
-         Criteria criteria = session.createCriteria(PSConfigStatus.class);
-         criteria.add(Restrictions.eq("statusId", statusID));
-         cfgStatus = (PSConfigStatus) criteria.uniqueResult();
+      CriteriaBuilder builder = session.getCriteriaBuilder();
+      CriteriaQuery<PSConfigStatus> criteria = builder.createQuery(PSConfigStatus.class);
```

Comment:
this is missing the criteria for statusId and is no longer returning a unique result.  Needs updated to include the criteria and return 1 result.